cookbook_se/recipes/native_typing/custom_objects.py

```diff
@@ -0,0 +1,38 @@
+"""
+This examples shows how users can serialize custom JSON'able dataclasses between successive tasks using the excellent
+@dataclasses_json library
+"""
+import typing
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+
+from flytekit import task, workflow
+
+
+@dataclass_json
+@dataclass
+class Datum(object):
+    x: int
+    y: str
+    z: typing.Dict[int, str]
+
+
+@task
+def stringify(x: int) -> Datum:
```

Comment:
is this called stringify because the Datum class gets rendered as a json string?

Reply:
naa, its just random name, was doing something else, will rework all examples at once. For now this is just to test

Reply:
can we name it something more descriptive then? i was confused at first :)